src/main/java/uk/gov/companieshouse/web/pps/controller/pps/EnterPPSDetailsController.java

```diff
@@ -51,6 +53,9 @@ public class EnterPPSDetailsController extends BaseController {
     private static final String ENTER_LFP_DETAILS_MODEL_ATTR = "enterLFPDetails";
```

Comment:
Should we keep this as LFP or we need to change this to PPS?

Reply:
Hmmm, good question Kenneth. I was leaning towards leaving it and then when we do some actual logic refactoring we could do it. i think that is why we left it out of the initial refactor.

Reply:
I think if we take away the mix of confusion with having both LFP and PPS here - this version could be simplified to be just `enterDetails` throughout controller / model / template?

Reply:
I didn't really see this as a refactoring ticker per se, but it does make sense to do that